sections/tooling/syntax-highlighting.js

```diff
@@ -7,7 +7,7 @@ const SyntaxHighlighting = () => md`
 
 This is what it looks like when properly highlighted:
 
-<img alt="Syntax highlighted styled component" src="http://imgur.com/k7h45c3.jpg" height="250px" />
+<img alt="Syntax highlighted styled component" src="https://imgur.com/k7h45c3.jpg" height="250px" />
```

Comment:
Sorry to ask! First of all, thank you for noticing! This must've slipped through the cracks. Even more so, because it doesn't make sense to host this on Imgur. Can you please help us to put it in the right place, and move it to our `static/` folder please 👍

Reply:
Sure thing!